app/views/listings/_listing.json.jbuilder

```diff
@@ -1,2 +1,4 @@
-json.extract! listing, :id, :type, :created_at, :updated_at
+json.extract! listing, :id, :name, :type
+json.created_at "#{listing.created_at.to_formatted_s(:long)} #{listing.created_at.zone}"
+json.updated_at "#{listing.updated_at.to_formatted_s(:long)} #{listing.updated_at.zone}"
```

Comment:
I guess now we have to have a time zone conversation. Should listings show up in time local to the request or to the viewer?

Reply:
LOVE that you thought of this! I would vote local to the viewer, but very open to whatever anyone else thinks.